### ISSUE TITLE
Error on CameraInit

### ISSUE THREAD

**Issue by GreenOpposite** [alicevision/meshroom#202]
My Canon EOS 750d got not recognized.
I was benable to fix the Problem by going into the _cameraSensors.db_ and changing
`Canon;Canon EOS Rebel T6i / 750D;22.3`
into
`Canon;Canon EOS 750D;22.3`

**Comment by simogasp** [comment]
yes the algorithm for searching inside the db has changed before the release and it is less flexible
see https://github.com/alicevision/AliceVision/issues/485
